app/MMB/src/appRegistry/ManageMyBookingPackage.js

```diff
@@ -7,6 +7,7 @@ import NavigationStack from '../navigation/NavigationStack';
 type Props = {|
   language: string,
   currency: string,
+  accessToken: string,
```

Comment:
Shouldn't we use context for this? I am not sure if prop-drilling for every `PrivateApiRenderer` is a good idea...

Reply:
Ok, also please try to fix the alignment of oneway/round/multicity arrows...